### ISSUE TITLE
Just a curiosity

### ISSUE THREAD

**Issue by mancho2000** [hsahovic/poke-env#326]
Hey everyone,

Do you mind sharing what elo is your agent and what does it play? I have been training some for random battles over the months without any decent results. Was curious of how others are doing :)

**Comment by SomeRandomGuy009** [comment]
try training in ou, in gen8 randbats, the nature is always random so kinda impossible to determine the exact stat

**Comment by SomeRandomGuy009** [comment]
mine reached about 1700 and currently is at 1600+

**Comment by mancho2000** [comment]
woa nice! much better than mine :) was that in randbats?

**Comment by SomeRandomGuy009** [comment]
> woa nice! much better than mine :) was that in ou?

yep, random battles doesn't yield good results because of "random natures"...maybe it would yield good output if the bot had a random_battle_sets.json file which it could look through every turn to assume the nature of a poke?

**Comment by akashsara** [comment]
@SomeRandomGuy009 what algorithm are you using to train your agent?

**Comment by SomeRandomGuy009** [comment]
> @SomeRandomGuy009 what algorithm are you using to train your agent?

well, i don't train agents...the algo i use for battles depends upon the bot's current elo. If it goes above 1500, predictions work quite often so it's minimax for that range. Below 1500, it's like simple heuristics.

**Comment by akashsara** [comment]
Oh are you using something like pmariglia's agent?

**Comment by SomeRandomGuy009** [comment]
> Oh are you using something like pmariglia's agent?

something like that but worse...it's like two agents..if elo goes above the specified no. it changes algo as specified.

**Comment by cameronangliss** [comment]
Closing due to inactivity. Feel free to make a discussion post in Discussions for more comms like these :)
